extensions/amp-video/0.1/amp-video.js

```diff
@@ -774,6 +777,7 @@ export class AmpVideo extends AMP.BaseElement {
     if (!captionsElement) {
       return;
     }
+    this.captionsAreDelegated_ = true;
```

Comment:
This architecture is wrong, you're mixing and tying up different concepts / features together, which is very hurtful long term. It becomes really hard to untangle as specific features evolve, or for new engs to understand.

What you mean by `captionsAreDelegated` is "we're using `amp-story-captions`.
What we want is `amp-story` components will toggle their captions on/off, this is not related to what system we use to render the captions.

Please use `this.isManagedByPool_()` to know if you're in a story context.